app/views/fragments/javascriptFirstSteps.scala.html

```diff
@@ -37,8 +38,12 @@
         touchpointBackendResolution <- touchpointBackendResolutionOpt;
         backend = touchpointBackendResolution.backend
     ) {
+    guardian.stripe = {
+        '@{UKStripeService.jsLookupKey}': '@backend.stripeUKMembershipService.publicKey.mkString',
+        '@{AUStripeService.jsLookupKey}': '@backend.stripeAUMembershipService.publicKey.mkString'
+    };
     guardian.stripeCheckout = {
-        key: '@backend.paymentService.stripeService.publicKey',
+        key: guardian.stripe['@{UKStripeService.jsLookupKey}'],
```

Comment:
Is something being hardcoded to use the main (UK) Stripe account here?

Reply:
As a fallback; it gets overridden in submit.js

Reply:
Ah I see - thanks